app/inbound/get-settlement-by-invoicenumber-and-value.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../data')
```

Comment:
files all good -- would probs change the filename to just all kebab case to match rest of service